src/hotspot/share/runtime/arguments.cpp

```diff
@@ -3232,7 +3232,7 @@ jint Arguments::finalize_vm_init_args(bool patch_mod_javabase) {
   UNSUPPORTED_OPTION(ShowRegistersOnAssert);
 #endif // CAN_SHOW_REGISTERS_ON_ASSERT
 
-  if (CRaCCheckpointTo && !os::Linux::prepare_checkpoint()) {
+  if (CRaCCheckpointTo && !crac::Linux::prepare_checkpoint()) {
```

Comment:
Does it really make sense to have `crac::Linux` instead of just `crac::`? The most of the operations are not related to Linux, so it may be even better to start abstracting them from Linux in this PR, to avoid another similar renaming in the same place.

Reply:
Ok, let's get rid of crac::linux now.